graylog2-web-interface/src/stores/lookup-tables/LookupTableDataAdaptersStore.js

```diff
@@ -10,7 +10,12 @@ const LookupTableDataAdaptersActions = ActionsProvider.getActions('LookupTableDa
 
 const LookupTableDataAdaptersStore = Reflux.createStore({
   listenables: [LookupTableDataAdaptersActions],
-
+  dataAdapter: null,
+  dataAdapters: undefined,
+  types: null,
+  pagination: null,
```

Comment:
Since we are setting the pagination in `init` and that only occurs once, when the store is initialized, I think we should move the initialization in here, what do you think?

Reply:
I think we cam do that too.